crates/polars-io/src/utils/other.rs

```diff
@@ -47,9 +47,9 @@ pub fn get_reader_bytes<'a, R: Read + MmapBytesReader + ?Sized>(
 ///
 /// # Safety
 /// The `out` vec outlives `bytes` (declare `out` first).
-pub unsafe fn maybe_decompress_bytes<'a>(
+pub fn maybe_decompress_bytes<'a, 'b: 'a>(
```

Comment:
learned new things about lifetimes recently 😁

Reply:
Then we can remove the `# Safety` comment as well. :+1:

Reply:
Ah, I removed - it doesn't show unless you go into "Files changed"

Reply:
Format 
Page